plugins/cue_points/quiz/services/QuizrReportingService.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Allows user to get reports regarding quizes
+ *
+ * @service quizReports
+ * @package plugins.quiz
+ * @subpackage api.services
+ */
+
+class QuizrReportingService extends KalturaBaseService
```

Comment:
Please revert this merge and add the report as new report type in the core reports service.